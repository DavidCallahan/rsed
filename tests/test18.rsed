$temp = "abbaxbbcbb"
split $temp with "b+"
print $0
print $1
print $2
print $3
print "next"
split $temp with "ab"
print $0
print $1
print $2

