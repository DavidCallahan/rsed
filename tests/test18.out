a
ax
c

next

baxbbcbb

